src/NServiceBus.RabbitMQ/Receiving/DelayedReceiver.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace NServiceBus.Transport.RabbitMQ
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using global::RabbitMQ.Client;
+    using global::RabbitMQ.Client.Events;
+
+    class DelayedReceiver
```

Comment:
I think `DelayedMessageReceiver` sounds decent. At this point I'm not sure this will even survive as a separate class vs. possibly trying to refactor the message pump code since these are are so similar.